src/common/models/Projection.ts

```diff
@@ -225,6 +226,10 @@ export class Projection {
     this.currentCaseDensity = lastValue(
       this.caseDensityRange.map(range => range && range.caseDensity),
     );
+    this.currentDailyDeaths =
+      this.currentCaseDensity === null
+        ? null
+        : CASE_FATALITY_RATIO * this.currentCaseDensity;
```

Comment:
@mikelehen do we have daily deaths that we can use? This is for the status text copy, where we show the number of daily deaths/100k and the number of daily cases/100k

> Over the last week, Mississippi has reported [10] new cases and [0.1] new deaths per day for every 100,000 residents.
>
> If these rates continue, we estimate less than 1-10% of Mississippi’s population will be infected in the next year.

For the cases, it's just the case density metric `currentCaseDensity`. I used `CASE_FATALITY_RATIO * currentCaseDensityByDeaths` first to calculate deaths, but in some cases (Benson County, ND) this is `0`, even though the case density is not zero (good catch @chasulin!).

![image](https://user-images.githubusercontent.com/114084/87453594-8d633e80-c5b7-11ea-99aa-e745b394bcc2.png)


See [CAN Phase 3 - Webapp Text](https://docs.google.com/document/d/1Jsj4zOksCJcgjQLUo3rwVAobEUkGLFukD1K71IqVNpo/edit#)

Reply:
I think this is normal / expected, right?  Benson County has cases but no deaths right now?  In general we expect cases to increase first, since more people will start getting sick, and then eventually a percentage of them may die.